optuna/logging.py

```diff
@@ -82,6 +82,39 @@ def get_logger(name: str) -> logging.Logger:
 def get_verbosity() -> int:
     """Return the current level for the Optuna's root logger.
 
+    Example:
+
+        Get the default verbosity level:
```

Comment:
A nit, but could you replace `:` with `.`?
```suggestion
        Get the default verbosity level.
```